code-challenges.js

```diff
@@ -26,17 +43,46 @@ const padres1984WorldSeriesRuns = [2, 5, 2, 2, 4]
 const padres1998WorldSeriesRuns = [6, 3, 5, 3]
 // Expected output: 17
 
-// Explain your code:
+//input: array of numbers
+//output: returns sum of the array of numbers
+//create a function that takes the array and adds each element together ex. 2+5+2+2+4 = 15 
+//iterate over the array with a for loop. 
 
-// --------------------2) Create a function that takes in a string and  returns the string with all the characters in reversed order. Use the test variable provided below.
+const addedRuns =(array) => {
+    let sum = 0
+    for (let i = 0; i < array.length; i++){
+        sum += array[i] //at first i just tried using + but it didnt work. remebered us using the += and then creating another variable of sum = 0
+        //return sum // i tried to return here first but it the output was 2 and 6. moved it outside the function and it did the trick
+    } return sum
+}
+    console.log(addedRuns(padres1984WorldSeriesRuns)) //output: 15
+    console.log(addedRuns(padres1998WorldSeriesRuns)) //output: 17
+// Explain your code: in the above example. I am creating a function that takes addedRuns. using a for loop to iterate through the array and adds the value of the current element of the array to sum. 
+
+// --------------------2) Create a function that takes in a string and returns the string with all the characters in reversed order. Use the test variable provided below.
 
 const lesMiserables = "Jean Valjean"
 // Expected output: "naejlaV naeJ"
 
 const lionKing = "Simba"
 // Expected output: "abmiS"
 
-// Explain your code:
+// input: a string
+// output: a string in reveresed order
+// create a function that turns the string into an array using .split
+// take in the new array string and use the method .reverse to reverse the words 
+// join the string back together using .join
+// return the new string
+
+const reveresedString= (string1) => {
+    let stringArr = string1.split("")// kept getting error because i had () with no "" in them
+    let newString = stringArr.reverse().join("")
+    return newString
+}//.join tried the join here and remebered i could do method after another methor on like 79 
+console.log(reveresedString(lesMiserables)) //output : naejlaV naeJ
+console.log(reveresedString(lionKing)) //output : abmiS
+
+// Explain your code: this one is fun! mark, seth and I came up with this one on thursday night! the code above takes the variable reveresedString and creates a function that turns the strings into an array. by doing this i created another variable that takes that array and reverses the order. then i joined them together to make a string again givin me the out out i wanted.
```

Comment:
I love that you are including notes on what did/didn't work and why!  This will help you move forward more quickly.  Woot!